app/bundles/AssetBundle/EventListener/AsssetExportListEventSubscriber.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\AssetBundle\EventListener;
+
+use Mautic\AssetBundle\Entity\Asset;
+use Mautic\AssetBundle\Event\AsssetExportListEvent;
+use Mautic\CoreBundle\Helper\PathsHelper;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+final class AsssetExportListEventSubscriber implements EventSubscriberInterface
```

Comment:
```suggestion
final class AssetExportListEventSubscriber implements EventSubscriberInterface
```
Looks like a small typo, should be two s letters, not three. Also, don’t forget to rename the file accordingly.

Reply:
fixed